snowfakery/__init__.py

```diff
@@ -1 +1,12 @@
+from pathlib import Path
+
 from .plugins import SnowfakeryPlugin, lazy  # noqa
+
+
+# TODO: when Python 3.6 is irrelevant, make this lazy:
+
+# https://www.python.org/dev/peps/pep-0562/
+
+version_file = Path(__file__).parent / "version.txt"
+with version_file.open() as f:
+    version = f.read().strip()
```

Comment:
This is fine, but you could have done `version = version_file.read_text().strip()`